app/controllers/locationOfGoods/EoriController.scala

```diff
@@ -48,7 +48,9 @@ class EoriController @Inject() (
 
   def onPageLoad(departureId: String, mode: Mode): Action[AnyContent] = actions.requireData(departureId) {
     implicit request =>
-      val preparedForm = request.userAnswers.get(EoriPage) match {
+      val preparedForm = request.userAnswers
+        .get(EoriPage)
+        .orElse(request.userAnswers.departureData.Consignment.LocationOfGoods.flatMap(_.EconomicOperator.map(_.toString))) match {
```

Comment:
can it just be .identificationNumber rather than .toString?
I see this is how It is in LocationOfGoodsAnswersHelper, think that is probably wrong too.